### PR TITLE
Add/fix information on nuxtChildKey

### PR DESCRIPTION
**Pull request by utatti**
This is the document-side pull request of https://github.com/nuxt/nuxt.js/pull/5516.

I've updated documentation on `nuxtChildKey` prop.

1. The default is not `$route.fullPath`. `$route.path` is used by default.
2. Documentation to handle internal `key` of `<router-view/>` is added.

FYI, I did the task 1 and 2 for EN, DE, ES, JA and KO.

- EN, DE, ES: They were written yet in English, so I just wrote in EN too.
- JA, KO: I can write JA and KO.

For the other languages, I did the task 1 because *fullPath by default* is wrong information and better be fixed soon.

cc: @potato4d and @DiyLecko

### CODE REVIEW

**Comment by clarkdo** [comment]
@Atinux Base branch should be changed since the pr is a breaking change.

**Comment by Atinux** [comment]
Well actually the documentation was wrong and the default value was not `$route.fullPath` 🙈

**Comment by Atinux** [comment]
@inouetakuya could you please resolve the conflicts for `ja/api/components-nuxt.md` please 🙏 ?

**Comment by potato4d** [comment]
@Atinux @inouetakuya
I noticed this PR now.
It can be solved by me!

**Comment by Atinux** [comment]
Thanks @potato4d :)

**Comment by inouetakuya** [comment]
@Atinux @potato4d 

> could you please resolve the conflicts for ja/api/components-nuxt.md please 🙏 ?

done 😄

**Comment by potato4d** [comment]
I'm sorry. I will do it today.

**Comment by potato4d** [comment]
@inouetakuya 
Thank you for doing it. sorry 🙇

File: zh/api/components-nuxt.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ description: 该组件用于在布局中显示页面组件（即非布局内容
 **Props**:
 - nuxtChildKey: `string`
   - 此prop将设置为`<router-view />`，可用于在动态页面和不同路径内进行转换。
-  - 默认: `$route.fullPath`
+  - 默认: `$route.path`
```
Comment (clarkdo) [inline zh/api/components-nuxt.md]: @warriorBrian Could you please help to add Chinese translation when you have time ?

**Pull request merged**